metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -264,6 +264,23 @@ void apply(final Hash hash, final String field, final String value) {
                 void apply(final Hash hash, final String field, final String value) {
                     hash.add(field, new Value(value));
                 }
+            },
+            /* For an indexed representation of arrays as hashes with 1, 2, 3 etc. keys.
+             * i.e. ["a", "b", "c"] as { "1":"a", "2":"b", "3": "c" }
+             * This is what is produced by JsonDecoder and Metafix itself for arrays.
```

Comment:
Where does Metafix do this?

Reply:
I meant the general behavior for repeated input fields, which become a list internally, and are emitted in that indexed form (see e.g. in [this test](https://github.com/metafacture/metafacture-fix/blob/4e26885f346b4cc3542c716661b4c3084ae12bae/metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java#L54)).

Reply:
I see, thanks for the clarification.